src/schemas/mini-program.ts

```diff
@@ -1,8 +1,9 @@
 export interface MiniProgramPayload {
     appid?          : string,    // optional, appid, get from wechat (mp.weixin.qq.com)
     description?    : string,    // optional, mini program title
-    pagepath?       : string,    // optional, mini program page path
-    thumbnailurl?   : string,    // optional, default picture, convert to thumbnail
+    pagePath?       : string,    // optional, mini program page path
+    thumbUrl?       : string,    // optional, default picture, convert to thumbnail
```

Comment:
This change will be a break change if you change the name of the properties.

Reply:
When developer updates `wechaty-puppet`, they should change the code for MiniProgram. 

The `mini-program.ts` in wechaty is useless now.

Reply:
For MiniProgram, I think the most important thing is the payload `MiniProgramPayload` in `wechaty-puppet` now.

Reply:
I understand, but the upgrade will break those users use `pagepath` and `thumbnailurl`, because you removed them.

Reply:
Agree, but if we should change the variable names be better, the developers should update too. It will be done sooner or later.

If they don't update `wechaty-puppet`, they can run everything ok. Right?

Reply:
You can add new properties there and keep the old ones, mark the old ones as `@deprecated`, and remove it in the future version.

Reply:
Understand!